.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug
+about: Something wrong with the project? Report it here!
+labels: 'bug'
+assignees: ''
+
+---
+
+**Description**
```

Comment:
Is this the standard template for Sigstore project?

If not I would think about having sections (beyond just description and version):
* What did you observe
* What was the expected outcome
* Steps to reproduce

Reply:
This is the standard template from https://github.com/sigstore/.github/tree/main/.github/ISSUE_TEMPLATE, I didn't want to change it. We could update the org-wide issue templates first?

Reply:
FWIW, the description asks for these details.